Note that MapView is not yet available on Android

benhoyt opened the pull request:
The MapView component is not labelled as iOS-only, but it is. It took me a bit of digging to figure out that the Android source code hasn't been released. Found it in this [SO answer](http://stackoverflow.com/questions/32625259/no-view-manager-defined-for-class-rctmap) in the end, and it is documented [here](http://facebook.github.io/react-native/docs/known-issues.html#views), but it'd be nice if it said that directly in the MapView docs. (The mention of the Android-specific prop `active` put me off the trail for a bit.)

facebook-github-bot-7 commented:
By analyzing the blame information on this pull request, we identified @nicklockwood and @jerolimov to be potential reviewers.

mkonicek commented:
Good idea but we should link to react-native-maps instead like in this documentation:
http://facebook.github.io/react-native/docs/known-issues.html#missing-android-modules-and-views

mkonicek commented on Libraries/Components/MapView/MapView.js:
```diff
@@ -303,6 +309,7 @@ const MapView = React.createClass({
     onAnnotationPress: React.PropTypes.func,
 
     /**
+     * Unused - Android version of MapView not currently available.
```
Engineers at Facebook can actually use the internal Android implementation and could get confused by this. Might be better to keep as is, given there's a big warning in the docs at the top (thanks for adding it!).

benhoyt replied:
Got it, will remove.

benhoyt commented:
Cool. How do I link in the source code docs? Markdown syntax links?

janicduplessis commented:
@benhoyt Yes markdown links will work.

You can test the website using
```
cd website
npm install
npm start
```

facebook-github-bot-2 commented:
@benhoyt updated the pull request.

benhoyt commented:
@mkonicek Both fixed now, thanks.

facebook-github-bot-4 commented:
@benhoyt updated the pull request.

benhoyt commented on Libraries/Components/MapView/MapView.js:
```diff
@@ -35,6 +35,15 @@ export type AnnotationDragState = $Enum<{
 }>;
 
+/**
+ * A component for displaying embeddable maps and annotations using the native
```
Fixed in 607f0ba

facebook-github-bot-9 commented:
@benhoyt updated the pull request.

facebook-github-bot-8 commented:
@benhoyt do you have any updates for this pull request? It's been a while since the last update so wanted to check in and see if you've looked at the requested changes.

benhoyt commented:
@satya164 @mkonicek What needs to happen here? I believe I addressed the issues and fixed the whitespace issue in https://github.com/facebook/react-native/commit/607f0ba213f40a5161ef08e2f15b72c0fcad83ff

janicduplessis commented:
Sorry we didn't get back to you. This looks good to me.

@facebook-github-bot shipit

facebook-github-bot-1 commented:
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/228558154179026/int_phab) to review.
